extraTs/utils/Validation.ts

```diff
@@ -0,0 +1,30 @@
+import validateEmail from './helpers'
+
+export default function validateUsers(users : IUsers[]) : void{
+    let countValid =0;
+    let countInvalid  =0;

```

Comment:
Write the return type of these variables

Reply:
i forgot to write the return type of this variable.